core/src/main/java/eu/kevin/core/plugin/Kevin.kt

```diff
@@ -1,12 +1,13 @@
 package eu.kevin.core.plugin
 
 import androidx.annotation.StyleRes
+import eu.kevin.core.R
 import java.util.*
 
 object Kevin {
 
     private var locale: Locale? = null
-    private var theme: Int = 0
+    private var theme: Int? = null
```

Comment:
why not just having a default value of `R.style.Theme_Kevin_Base`?

Reply:
good question, fixed